### DISCUSSION TITLE
Readings in prep for week 7

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#9] [Reading responses etc]
After reading Montfort on "[Standard Starting Points](https://mitpress.ublish.com/ebook/epah2e-preview/12629/C1)" and D'Ignazio and Klein on why "[What Gets Counted Counts](https://ebookcentral.proquest.com/lib/pitt-ebooks/detail.action?docID=6120950)" (and why that's not _all_ that counts), and watching [Beshero-Bondar](https://cmu-lib.github.io/dhlg/project-videos/besherobondar/) and [Brown](https://cmu-lib.github.io/dhlg/project-videos/brown/) on how they organize and process things in the real world into datasets, please write a short response to let us know where your mind went. For example, you might...

* Highlight new ideas or key terms, especially those that you expect will be useful in thinking about digital projects.
* Highlight sentences, passages, or methods that struck you as especially well done, perhaps worth imitating.
* Link outward to something a passage reminds you of, explaining where you're leading us.
* Pose questions to your classmates about a passage, especially (but not exclusively) if something is confusing.
* Respond to any of the above – which you can now do directly, using threads! Joy.

Where possible, please try to point us to at least one particular passage from the text you're responding to – quoting if you can, giving a timestamp or page reference either way – as a way to help keep our discussion grounded.

**Comment by chloe-cmu** [comment]
I was really intrigued by AmyJo Brown's talk - and especially her mention of the possibility of using data dictionaries and data diaries. I have to confess that I haven't been keeping very careful notes about my project thus far besides the basic timekeeping in the mindfulness journal - it doesn't seem too wildly complicated, it's just the one dataset, I'm not working with anyone else, and I do have the whole Undo/Redo thing, after all  -- but then earlier today I sat down with a friend to discuss the data, and she asked me (relatively basic) questions I had to think about for a while or go spelunking around in my files in order to anwer. That surprised me! I had been working with the data less than a week before. The possibility of having a data dictionary and a data diary had never even occured to me, but I'll start doing that next week. It's also good to remember the principles discussed in Data Feminism as I'm constructing my own little dataset. Every kind of counting and categorization comes with its own limits, and limitations. I think I might start trying to keep a list of the things that my dataset can't show me or that I know it will miss, just to keep a close eye on what I can actually do with this project.

**Comment by amj169** [comment]
This chapter of Data Feminism was a great tool to remind me to think beyond just the medium of how I’d like to represent my public facing data. For me, data visualization can sometimes take a back seat when there are so many other moving parts. Certainly not because it isn’t important, but rather because of the order my research sequencing often seems to take. It’s easy for me to privilege data collection ahead of data viz because of the chronological way I handle such tasks. Considering the role data viz will play goes hand in hand with the reasons for collecting the data in the first place, so I’m reminded to zoom in on that in a more simultaneous manner rather than treating it as a separate  'last step' entity. 

Additionally, this chapter makes me think about routine experiences I’ve had where the purpose for collection and context of the collector become  problematic and blurred. I can think of several (Dr. visits, auto insurance forms, the DMV in general) but I’ll offer my thoughts on a recent incident I had with the census since they discuss census data in this chapter: 

Over the summer a woman who was employed by the US census came to our door to collect census information from us. Somewhere between politely assuring her we would just fill it out online to get her to leave, and getting roped into filling it out in-person on my porch, the woman who was collecting the information began to reveal her political stance through several out-of-pocket comments and the recommendation of a right-wing podcast to me. After this interaction, I felt a lot less safe sharing this information with a person who just declared her supports of a very open anti LGBTQ+ spokesperson than I would have just filling out the form online. If someone working with census data sees a queer person on paper, they probably won’t come find them if they had an issue, but in this case, even sharing data that normally wouldn’t have put me at risk (most likely), opened up that possibility because this person knew where she could find me. There was this unnecessary personal addition that quickly became problematic. 

This is a case where we can analyze the system of collection through D’Ignazio and Klein’s feminist lens. The census created an opportunity to dispatch people on the ground to collect information in person to gather more data. Perhaps people ignored the census, didn’t want to fill it out for a number of reasons including putting them at risk, didn’t have access to the tools to complete it, etc., so the idea was that by doing this they could streamline the process by this rhetorical action of manual data collection to get more responses. However, this only exacerbates many of the issues with census data collection in the first place, especially for people who are a part of underrepresented and disenfranchised groups to begin with. Now you’ve added a measure that arguably puts specific groups in more danger than if they just ignored the notices in the first place or opted to fill it out electronically. We can also ask questions about the training system in place? What kind of training did employees receive? Did they receive DEI training? Is there a policy where they aren’t supposed to discuss politics with people? Were they simply advised to be personable, so they get a larger buy-in for people to comply? Is their job quota driven?  Upon brief analysis of this system, there are serious implications that adversely affect certain groups over others, much like the TSA gender binary example.

**Reply by Khushboobhutani** [reply]
I am sorry to hear that you had to undergo such an experience. This instance does open up a fertile space to discuss varied practices with respect to data collection and the kind of questionnaires built for such purposes. It also makes me think if the questions differ from person to person, or with every community. Would these efforts help the marginalized gain more representation and empower them or will it further cement the differences and make people more skeptical of such practices?

**Comment by Khushboobhutani** [comment]
Elisa Beshero-Bondar and AmyJo Brown’s videos provide interesting methodologies to think about our research projects. The ecosystem of data that Beshero-Bondar describes can be an engaging way to build connections across different kinds of data sets to create an ecosystem. I am especially intrigued by her reading and the visual translation of data from Robert Southey’s poem. The encoding of mappable and mythical places to study the textual hierarchy of the poem in the form of network graphs was unprecedented for me. Both Beshero and Brown appeal for creating different forms of data organization and asking right questions to categorize the data well, a trait whose lack I can feel in my project at this point. In continuation with the process of data collection, Lauren F. Klein and Catherine D'lgnazio's chapter questions the commonplace approach to data design and utilization, and provides much needed insights into the unequal power structures in the realm of data. It was interesting to note and ponder upon the proposition that data ethics and its focus on fairness and biases often ends up creating structures that protect power. The piece provides an interesting theoretical ground to think through concerns surrounding the generation, dissemination, representation and consumption of data in a data driven society like ours.

**Comment by SidraNArshad** [comment]
Near the end of Data Feminism’s fourth chapter, there is a short discussion of the ways in which black activists and their contribution to the dismantling of slavery has been overlooked in history. As this chapter discusses, such an overlooking and erasure of their labor partly contributes to the white savior narrative that the recordings of many white abolitionists work has established. The Colored Conventions Project was created in order to rectify this erasure, and such a project relates to the goals of my own DSAM project. I know that my project isn’t being created on the grounds of subverting any white savior narrative, but it is attempting to bring light to an overlooked issue affecting black activists today. The notion of recognition continues to exist as a tension that perpetuates discriminatory trends.

**Comment by Rezaetmd** [comment]
The second reading captivated me more, discussing instances of tangible outcomes and biases stemming from classification systems, like those found in the US Census and airport security screenings. The author highlights how airport security often employs gender-based classification for risk assessment, potentially resulting in discriminatory practices and profiling, particularly affecting transgender and gender non-conforming individuals outside conventional gender norms. I personally didn't know that such gender biases could exist in places that we don't expect them. Also, I like @Khushboobhutani found the two videos interesting in that it talks about 1)how we should approach data sets to stay organized throughout our projects and 2) how they ended up going back to the collected data to check if there is any discrepancies between what the software has OCRed and what was actually written in each form.
